doc/functions/overrides.xml

```diff
@@ -86,11 +86,11 @@ helloWithDebug = pkgs.hello.overrideAttrs (oldAttrs: rec {
     in this case, as it overrides only the attributes of the final derivation.
     It is for this reason that <varname>overrideAttrs</varname> should be
     preferred in (almost) all cases to <varname>overrideDerivation</varname>,
-    i.e. to allow using <varname>sdenv.mkDerivation</varname> to process input
+    i.e. to allow using <varname>stdenv.mkDerivation</varname> to process input
     arguments, as well as the fact that it is easier to use (you can use the
     same attribute names you see in your Nix code, instead of the ones
     generated (e.g. <varname>buildInputs</varname> vs
-    <varname>nativeBuildInputs</varname>, and involves less typing.
+    <varname>nativeBuildInputs</varname>), and it involves less typing).
```

Comment:
has another typo appeared here with the extra `)`?

Reply:
I don't believe so: before, the parentheses were not balanced. There were two opening parentheses and no closing ones.

Reply:
I think the PR is correct, the last paren closes the paren starting at "(you can use the ..." above. Parens within parens is perhaps not the best form, however.